src/main/java/org/opentripplanner/middleware/otp/OtpDispatcher.java

```diff
@@ -120,4 +125,43 @@ private static OtpDispatcherResponse sendOtpRequest(
                 bodyContent);
         return new OtpDispatcherResponse(otpResponse);
     }
+
+    public static OtpResponse sendOtpRequestWithErrorHandling(String sentParams) {
+        return handleOtpDispatcherResponse(() -> sendOtpPlanRequest(OtpVersion.OTP1, sentParams));
+    }
+
+    public static OtpResponse sendOtpRequestWithErrorHandling(OtpRequest otpRequest) {
+        return handleOtpDispatcherResponse(() -> sendOtpPlanRequest(OtpVersion.OTP1, otpRequest));
+    }
+
+    private static OtpResponse handleOtpDispatcherResponse(Supplier<OtpDispatcherResponse> otpDispatcherResponseSupplier) {
+        OtpDispatcherResponse otpDispatcherResponse;
+        try {
+            otpDispatcherResponse = otpDispatcherResponseSupplier.get();
+        } catch (Exception e) {
+            BugsnagReporter.reportErrorToBugsnag(
+                "Encountered an error while making a request ot the OTP server.",
```

Comment:
"to" not "ot".

Reply:
Carry over of a prior typo... fixed in https://github.com/ibi-group/otp-middleware/pull/240/commits/47f339183f5cd76606d0e6836546bc2033ca9d08.